### DISCUSSION TITLE
S905X4 kernel and dtb for openwrt

### DISCUSSION THREAD

**Discussion by lspvic** [devmfc/debian-on-amlogic#22] [Q&A]
I apologize for mentioning OpenWRT. 
I have an HK1 RBox X4 with s905x4_4g_1gbit and I'm trying to install OpenWRT on it. I've been struggling to combine different kernels and DTBs, but none of them seem to work. 
After numerous attempts, I discovered that the image for x96 X4 from your repository works flawlessly with my HK1 RBox X4.
I've tried using the DTB (and also the kernel-6.6.8) from the x96 X4 for my MyBox, along with the OpenWRT rootfs, but all of these attempts have failed.
Could you please share the kernel and DTB sources for the x96 X4 1Gbit. I'd like to rebuild them with the OpenWRT source code. Thank you very much~

**Comment by devmfc** [comment]
I will in time, when I have time.... See my Linux repo, there are branches where I started to commit the changes for the S4, not for the SC2 yet.  But I don't have time now or in the next few week. Sorry, this is just a hobby project for me.

**Comment by zouyonghao** [comment]
Thank you, @lspvic @devmfc, for this post and this amazing project. I've successfully booted my HK1 RBox X4 with Debian and set up an OpenWRT docker!

**Reply by songyuyx** [reply]
> Thank you, @lspvic @devmfc, for this post and this amazing project. I've successfully booted my HK1 RBox X4 with Debian and set up an OpenWRT docker!

Dear Sir,  may I have your image download link of "HK1 RBox X4 with Debian ".  I would like to do same thing as you did

**Reply by zouyonghao** [reply]
> > Thank you, @lspvic @devmfc, for this post and this amazing project. I've successfully booted my HK1 RBox X4 with Debian and set up an OpenWRT docker!
> 
> Dear Sir, may I have your image download link of "HK1 RBox X4 with Debian ". I would like to do same thing as you did

I just use the official image
